pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -72,12 +68,16 @@ public void map(InfoMap infoMap) {
         torch.sharedMap(infoMap);
 
         infoMap
-            .put(new Info("basic/containers").cppTypes("c10::optional"))
-
             .put(new Info().enumerate().friendly())
             .put(new Info().javaText("import org.bytedeco.pytorch.*;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.cuda.functions.*;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.Error;"))
+            .put(new Info().javaText("import org.bytedeco.pytorch.helper.*;"))
+            .put(new Info().javaText("import org.bytedeco.cuda.cudart.*;"))
+            .put(new Info().javaText("import org.bytedeco.cuda.cusparse.*;"))
+            .put(new Info().javaText("import org.bytedeco.cuda.cublas.*;"))
+            .put(new Info().javaText("import org.bytedeco.cuda.cusolver.*;"))
+            .put(new Info().javaText("import org.bytedeco.cuda.cudnn.*;"))
+            // .put(new Info().javaText("import org.bytedeco.cuda.nccl.*;")) // Not on Windows
+            .put(new Info().javaText("import org.bytedeco.pytorch.chrono.*;"))
```

Comment:
Instead of doing that, please add cudnn.class to the `@Platform(inherit=...` list.

Reply:
Done for cuda, but what `helper.class` and `chrono.class` are you talking about ?

Reply:
Right, JavaCPP assumes the helper classes to be in the same package as the generated classes, so let's remove that package too?

As for chrono, see https://github.com/bytedeco/javacpp/pull/766#issuecomment-2264430580